src/amber/cli/templates/app/src/controllers/error_controller.cr.ecr

```diff
@@ -1,18 +0,0 @@
-# Error Controller
-# Ensure Amber::Pipe::Error.new is inside config/router.cr pipelines
-# Put error templates inside src/views/error/
-class Amber::Controller::Error < Amber::Controller::Base
-  LAYOUT = "application.<%= @language %>"
-
-  # def forbidden
-  #   render("403.<%= @language %>")
-  # end
-
-  # def not_found
-  #   render("404.<%= @language %>")
-  # end
-
-  # def internal_server_error
-  #   render("500.<%= @language %>")
-  # end
-end
```

Comment:
We do not want this generated by default for `amber new app` anymore, correct?

Reply:
correct.  This is to remove that since it isn't fully functional.